.claude/crates/api.md

```
@@ -18,10 +18,12 @@ Thin axum REST + WebSocket server — entry point for external clients.
 - WebSocket message types are defined in `models` — changing them is a breaking API change.
 
 ## Relations
-- Depends on nebula-engine, nebula-storage, nebula-credential, nebula-execution. Highest layer — nothing depends on it.
+- Depends on nebula-storage, nebula-workflow, nebula-action, nebula-plugin. Highest layer.
 
-<!-- reviewed: 2026-03-30 — derive Classify migration -->
-
-<!-- reviewed: 2026-04-02 -->
+## Traps
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)